pkg/virt-launcher/virtwrap/network/nichotplug.go

```diff
@@ -49,10 +49,11 @@ type virtIOInterfaceManager struct {
 }
 
 const (
-	libvirtInterfaceLinkStateDown = "down"
+	libvirtInterfaceLinkStateDown         = "down"
+	affectDeviceLiveAndConfigLibvirtFlags = libvirt.DOMAIN_DEVICE_MODIFY_LIVE | libvirt.DOMAIN_DEVICE_MODIFY_CONFIG
```

Comment:
Please add a note in the commit message about the need to copy it here in order to avoid cyclic dependency.